examples/facial-recognition.ipynb

```diff
@@ -173,7 +177,7 @@
    "outputs": [],
    "source": [
     "# Fill here the bucket name you created in COS Dashboard \n",
-    "BUCKET = 'pywren-bucket-for-*****'"
+    "BUCKET = 'pywren-bucket-for-richh-2'"
```

Comment:
Did you mean to change this? I like things like '******' to make it clear that it is a placeholder, also easy to find.

Reply:
good catch - fixed